src/Response.php

```diff
@@ -1031,7 +1031,7 @@ public function getIsEmpty()
      * The default implementation will convert [[data]] into [[content]] and set headers accordingly.
      * @throws InvalidConfigException if the formatter for the specified format is invalid or [[format]] is not supported
      */
-    protected function prepare()
+    public function prepare()
```

Comment:
The API with this change doesn't look good. How about calling it in a getter instead?